lib/cipherstash/index/bloom_filter.rb

```diff
@@ -0,0 +1,87 @@
+require_relative "../client/error"
```

Comment:
Feels a little weird to reach over into the `client` dir here. Any feels on this @mpalmer?

Reply:
You're right, it does feel weird.  If you're up for creating a separate `Index::Error` class hierarchy, I would not be against it, although at the same time if you feel that it's more hassle than is warranted, we could defer it into a "make the StashRB error hierarchy less of a dog's breakfast" hygiene card for the future.

Reply:
Added a card to Notion.